src/Sarif.Multitool/SarifPropertyName.cs

```diff
@@ -6,17 +6,27 @@ namespace Microsoft.CodeAnalysis.Sarif.Multitool
     public static class SarifPropertyName
     {
         public const string AnalysisTarget = "analysisTarget";
+        public const string AnalysisToolLogFiles = "analysisToolLogFiles";
```

Comment:
>AnalysisToolLogFiles [](start = 28, length = 20)

A bunch of new properties I have to visit to fully implement the restored rules in SARIF v2.